keyboards/nullbitsco/nibble/serial.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
```

Comment:
We have the `/drivers/avr/serial.c`, and depending on what you're trying to do, you may be able to use that.

Reply:
That's software serial. It would probably be better to use `tmk_core/common/uart.c` or `tmk_core/protocol/serial_uart.c`.

Reply:
Thanks for the heads up! I didn't realize those changes had been included in `tmk_core/common/uart.c`. I'll refactor to leverage that code instead of my own.